src/app/lib/config/services/korean.js

```diff
@@ -122,40 +122,32 @@ const service = {
   footer: {
     externalLink: {
       href: 'https://www.bbc.co.uk/help/web/links/',
-      text: 'Read about our approach to external linking.',
+      text: '외부 콘텐츠 링크에 대한 본사 정책 보기',
```

Comment:
```suggestion
      text: '외부 콘텐츠 링크에 대한 본사 정책 보기.',
```

It looks like on the other `externalLink.text` values there is a trailing full stop?